src/js/jsonschema-app.js

```diff
@@ -0,0 +1,93 @@
+// this JavaScript snippet stored as src/js/jsonschema-app.js
+function App() {
+  // this JavaScript snippet is later referred to as <<jsonschema-app>>
+  const schema = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://nlesc-jcer.github.io/cpp2wasm/NNRequest.json",
+    "type": "object",
+    "properties": {
+      "epsilon": {
+        "title": "Epsilon",
+        "type": "number",
+        "minimum": 0
+      },
+      "guess": {
+        "title": "Initial guess",
+        "type": "integer",
+        "minimum": -100,
+        "maximum": 100
+      }
+    },
+    "required": ["epsilon", "guess"],
+    "additionalProperties": false
+  }
+  // this JavaScript snippet is appended to <<jsonschema-app>>
+  const Form = JSONSchemaForm.default;
+  const uiSchema = {
+    "guess": {
+      "ui:widget": "range"
```

Comment:
Much nicer if we have a number field instead of range.
If you remove line 28, it will automatically become a number field.

Reply:
This was done to show off the uiSchema customizing in http://localhost:8000/#/?id=json-schema-powered-form. 

Do you want to remove the slider in the plot and in the json schema form examples? Or just use a number field in the plot example?